docs/simulation/state-merging.md

```diff
@@ -0,0 +1,14 @@
+# State merging
+In the game, it may happen that two different countries own provinces that are part of a same state. Suppose a situation where a country acquires the remainder of a state it already owned. What happens to the buildings?
+
+## Ports
+The port rules are straightforward. The highest-level port is retained, and the rest are deleted.
+
+## Factories
+Factory logic is quite complicated. If there are two factories of the same type in the states to be merged, the following rules are applied in order:
+
+- If the factory in the state that was already owned is subsidized, it takes priority, regardless of level.
```

Comment:
When I tried, the existing factory always stays regardless of subsidies, hiring priority or level.